mail_service/worker_notify_user.py

```diff
@@ -100,8 +100,8 @@ def __init__(self, job_info):
             logger.debug("Processing mail for failed build.")
             self.image_under_test = job_info.get("project_name")
 
-        self.project = self.job_info["namespace"] + "/" + \
-            self.job_info["job_name"]
+        self.project = self.job_info["namespace"].split("-")[0] + \
+            "/" + self.job_info["job_name"]
 
```

Comment:
Hmm i don't a user would and should be interested in the name we give to the project internally. What is he going to do with that information anyway. He is only interested in the name of the container with which he can pull the image, and that should be unique enough as it includes the full name, including tag.